components/dada-parse/src/parser/code.rs

```diff
@@ -312,13 +312,13 @@ impl CodeParser<'_, '_> {
         } else if let Some((id_span, id)) = self.eat(Identifier) {
```

Comment:
We are going to want to support `-` and `+` as unary operators, I imagine, so that we can parse this:

```
x = -5
y = -(1)
z = -x
```

We might want to do something mildly unconventional and parse `-5` without any whitespace as a literal that is explicitly a signed integer.

Reply:
Well, we could code it up either way. I'm honestly not sure if it *should* be parsed as a binary operator! It sure *looks* like a missing comma to me. =)